Integrations/Securonix/Securonix.py

```diff
@@ -888,6 +1023,76 @@ def get_watchlist(client: Client, args) -> Tuple[str, Dict, Dict]:
     return human_readable, entry_context, watchlist
 
 
+def create_watchlist(client: Client, args) -> Tuple[str, Dict, Dict]:
+    """Create a watchlist.
+
+    Args:
+        client: Client object with request.
+        args: Usually demisto.args()
+    Returns:
+        Outputs.
+    """
+    watchlist_name = args.get('watchlist_name')
+
+    response = client.create_watchlist_request(watchlist_name)
+
+    if 'successfully' not in response:
+        raise Exception(f'Failed to list watchlists.')
```

Comment:
maybe add response in the Exception? or is there a reason? need to see how dose it look.